<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedPlus - Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #f5f7fa, stop:1 #c3cfe2);
}</string>
  </property>

  <!-- Decorative Background Circle 1 -->
  <widget class="QFrame" name="frameCircle1">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>-80</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(33, 147, 176, 0.2), stop:1 transparent);
    border-radius: 150px;
    border: none;
}</string>
   </property>
  </widget>

  <!-- Decorative Background Circle 2 -->
  <widget class="QFrame" name="frameCircle2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qradialgradient(cx:0.5, cy:0.5, radius:1,
        fx:0.5, fy:0.5, stop:0 rgba(109, 213, 237, 0.2), stop:1 transparent);
    border-radius: 175px;
    border: none;
}</string>
   </property>
  </widget>

  <!-- Header Label -->
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>40</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>4</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 36px;
    font-weight: bold;
    color: #ffffff;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:0.5 #42a5f5, stop:1 #64b5f6);
    padding: 20px 30px;
    border-radius: 15px;
    border: 3px solid rgba(255, 255, 255, 0.3);
    letter-spacing: 2px;
    qproperty-alignment: AlignCenter;
}</string>
   </property>
   <property name="text">
    <string>üìù Registration Form</string>
   </property>
  </widget>

  <!-- Subtitle -->
  <widget class="QLabel" name="lblSubtitle">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>155</y>
     <width>500</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #37474f;
    background: transparent;
    qproperty-alignment: AlignCenter;
    font-weight: 500;
}</string>
   </property>
   <property name="text">
    <string>Create your MedPlus account to get started</string>
   </property>
  </widget>

  <!-- Form Container Frame -->
  <widget class="QFrame" name="frameForm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>550</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: rgba(255, 255, 255, 0.95);
    border-radius: 20px;
    border: 2px solid rgba(25, 118, 210, 0.3);
}</string>
   </property>
  </widget>

  <!-- Full Name Label -->
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üë§ Full Name:</string>
   </property>
  </widget>

  <!-- Full Name Input -->
  <widget class="QLineEdit" name="fullNameLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>255</y>
     <width>300</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 15px;
    color: #2c3e50;
    padding: 10px 15px;
    border: 2px solid #1976d2;
    border-radius: 10px;
    background-color: white;
}

QLineEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QLineEdit:focus {
    border: 2px solid #0d47a1;
    background-color: white;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter your full name</string>
   </property>
  </widget>

  <!-- Username Label -->
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>325</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üÜî Username:</string>
   </property>
  </widget>

  <!-- Username Input -->
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>300</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 15px;
    color: #2c3e50;
    padding: 10px 15px;
    border: 2px solid #1976d2;
    border-radius: 10px;
    background-color: white;
}

QLineEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QLineEdit:focus {
    border: 2px solid #0d47a1;
    background-color: white;
}</string>
   </property>
   <property name="placeholderText">
    <string>Choose a username</string>
   </property>
  </widget>

  <!-- Password Label -->
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üîí Password:</string>
   </property>
  </widget>

  <!-- Password Input -->
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>385</y>
     <width>300</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 15px;
    color: #2c3e50;
    padding: 10px 15px;
    border: 2px solid #1976d2;
    border-radius: 10px;
    background-color: white;
}

QLineEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QLineEdit:focus {
    border: 2px solid #0d47a1;
    background-color: white;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Create a strong password</string>
   </property>
  </widget>

  <!-- Confirm Password Label -->
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>455</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üîê Confirm:</string>
   </property>
  </widget>

  <!-- Confirm Password Input -->
  <widget class="QLineEdit" name="confirmPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>300</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 15px;
    color: #2c3e50;
    padding: 10px 15px;
    border: 2px solid #1976d2;
    border-radius: 10px;
    background-color: white;
}

QLineEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QLineEdit:focus {
    border: 2px solid #0d47a1;
    background-color: white;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Re-enter password</string>
   </property>
  </widget>

  <!-- Email Label -->
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üìß Email:</string>
   </property>
  </widget>

  <!-- Email Input -->
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>515</y>
     <width>300</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 15px;
    color: #2c3e50;
    padding: 10px 15px;
    border: 2px solid #1976d2;
    border-radius: 10px;
    background-color: white;
}

QLineEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QLineEdit:focus {
    border: 2px solid #0d47a1;
    background-color: white;
}</string>
   </property>
   <property name="placeholderText">
    <string>your.email@example.com</string>
   </property>
  </widget>

  <!-- Phone Number Label -->
  <widget class="QLabel" name="lblPhone">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>585</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    font-weight: 600;
    color: #1565c0;
    background: transparent;
    padding: 4px;
    letter-spacing: 0.5px;
}</string>
   </property>
   <property name="text">
    <string>üì± Phone:</string>
   </property>
  </widget>

  <!-- Phone Input -->
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>300</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 15px;
    color: #2c3e50;
    padding: 10px 15px;
    border: 2px solid #1976d2;
    border-radius: 10px;
    background-color: white;
}

QLineEdit:hover {
    border: 2px solid #1565c0;
    background-color: #f5f9ff;
}

QLineEdit:focus {
    border: 2px solid #0d47a1;
    background-color: white;
}</string>
   </property>
   <property name="placeholderText">
    <string>+91 XXXXX XXXXX</string>
   </property>
  </widget>

  <!-- Register Button -->
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>665</y>
     <width>140</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 18px;
    font-weight: bold;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:1 #42a5f5);
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1565c0, stop:1 #1976d2);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #0d47a1, stop:1 #1565c0);
}</string>
   </property>
   <property name="text">
    <string>‚úì Register</string>
   </property>
  </widget>

  <!-- Login Button -->
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>665</y>
     <width>140</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 18px;
    font-weight: bold;
    color: #1976d2;
    background: transparent;
    border: 2px solid #1976d2;
    border-radius: 12px;
    padding: 12px 24px;
}

QPushButton:hover {
    background-color: #e3f2fd;
    border-color: #0d47a1;
}

QPushButton:pressed {
    background-color: #bbdefb;
}</string>
   </property>
   <property name="text">
    <string>‚Üê Login</string>
   </property>
  </widget>

  <!-- Cancel Button -->
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>665</y>
     <width>140</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 18px;
    font-weight: bold;
    color: #757575;
    background: transparent;
    border: 2px solid #bdbdbd;
    border-radius: 12px;
    padding: 12px 24px;
}

QPushButton:hover {
    background-color: #f5f5f5;
    border-color: #757575;
}

QPushButton:pressed {
    background-color: #e0e0e0;
}</string>
   </property>
   <property name="text">
    <string>‚úó Cancel</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>